cubano-webdriver-manager/src/main/java/org/concordion/cubano/driver/web/provider/LocalBrowserProvider.java

```diff
@@ -100,7 +100,7 @@ protected void setupBrowserManager(WebDriverManager instance) {
 
         instance.setup();
 
-        driverPath = instance.getBinaryPath();
+        driverPath = instance.getDownloadedDriverPath();
```

Comment:
@andrew-sumner @jimmykemp This appears to be the closest method now. Please check. Thanks :)